evalml/pipelines/components/transformers/imputers/per_column_imputer.py

```diff
@@ -79,14 +77,14 @@ def transform(self, X, y=None):
             ww.DataTable: Transformed X
         """
         X_ww = infer_feature_types(X)
-        X = _convert_woodwork_types_wrapper(X_ww.to_dataframe())
-        X_t = X.copy()
+        original_logical_types = {col: value for col, value in X_ww.ww.logical_types.items()}
+
         cols_to_drop = []
         for column, imputer in self.imputers.items():
-            transformed = imputer.transform(X[[column]]).to_dataframe()
+            transformed = imputer.transform(X_ww[[column]])
             if transformed.empty:
                 cols_to_drop.append(column)
             else:
-                X_t[column] = transformed[column]
-        X_t = X_t.drop(cols_to_drop, axis=1)
-        return _retain_custom_types_and_initalize_woodwork(X_ww, X_t)
+                X_ww.ww[column] = transformed[column]
+        X_t = X_ww.ww.drop(cols_to_drop)
```

Comment:
`axis` defaults to 0 [in pandas](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.drop.html). Is that important here?

Reply:
The `drop` method in the woodwork accessor does not have an `axis` parameter. It only drops columns.